src/fiatExchanges/quotes/ExternalQuote.ts

```diff
@@ -92,15 +93,28 @@ export default class ExternalQuote extends NormalizedQuote {
     return null
   }
 
+  getReqsSubtitle(): string | null {
+    return !isSimplexQuote(this.quote) &&
+      this.getPaymentMethod() === PaymentMethod.Airtime &&
+      this.quote.extraReqs
+      ? this.quote.extraReqs.mobileCarrier ?? ''
+      : this.getKycInfo()
+  }
+
   getKycInfo(): string | null {
     return strings.idRequired
   }
 
   getTimeEstimation(): SettlementEstimation {
-    // payment method can only be bank or card
-    return this.getPaymentMethod() === PaymentMethod.Bank
-      ? DEFAULT_BANK_SETTLEMENT_ESTIMATION
-      : DEFAULT_CARD_SETTLEMENT_ESTIMATION
+    // payment method can be bank, card or airtime
+    switch (this.getPaymentMethod()) {
+      case PaymentMethod.Bank:
+        return DEFAULT_BANK_SETTLEMENT_ESTIMATION
+      case PaymentMethod.Airtime:
+        return DEFAULT_AIRTIME_SETTLEMENT_ESTIMATION
+      default:
+        return DEFAULT_CARD_SETTLEMENT_ESTIMATION
```

Comment:
seems like the default should be bank, since it's the most conservative of these

Reply:
actually, this is better:
- define a constant with type Record<PaymentMethod, SettlementTimeEstimation>
- use that to map payment method to settlement time

that way, if we add a payment method but forget to update this, we get a compile-time error instead of silently defaulting to another payment method's time estimation

Reply:
Settlement times are only defined for Bank, Card and Airtime, however PaymentMethod also includes Coinbase, MobileMoney and FiatConnectMobileMoney, should I have a default settlement for these other payment methods? Otherwise the map doesn't work because for key type PaymentMethod there are missing values.

Reply:
(discussed offline)